fantasy_postseason_challenge/dashboard.py

```diff
@@ -17,15 +17,27 @@
 def logged_in_homepage():
     league_memberships = current_user.memberships
     team_names = []
+    league_ids = []
+    commissioner_leagues = []
+    commissioner_league_ids = []
+    
     for league_membership in league_memberships:
         for member in league_membership.member_list:
             if member.account_id == current_user.id:
                 team_names.append(member.team_name)
+                break
+        if league_membership.commissioner_id == current_user.id:
+            commissioner_league_ids.append(league_membership.id)
```

Comment:
make a list of league objects instead of two lists of league data. then in the html page just use league.id and league.name

Reply:
v smart